### PR TITLE
appcenter-codepush-release-react-native 0.0.3

### PR DESCRIPTION
**Pull request by adrianchinghc**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by tadam313** [comment]
hi @adrianchinghc! Thanks for the contribution! It looks like we have a merge conflict in this PR, can you please update your fork with the latest changes from this repository and re-trigger the build? Much appreciated 🙇

File: steps/appcenter-codepush-release-react-native/0.0.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  react_native_project_root: $REACT_NATIVE_PROJECT_ROOT
+- app_id: $APPCENTER_APP_ID
+  opts:
+    description: |
+      AppCenter Application Identifier is a combination of your Organization/Owner Name and an Application Name.
+      For example: `Upstack-Studio/bitrise-android`
+    is_required: true
+    summary: AppCenter application identifier (Organization/Owner Name + App Name).
+    title: AppCenter App ID
+- api_token: $APPCENTER_API_TOKEN
```
Comment (tadam313) [inline steps/appcenter-codepush-release-react-native/0.0.3/step.yml:47]: isn't it a sensitive input variable? 🤔  if so please follow the section here: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#secret-environment-variables-in-steps

**Review by tadam313 — CHANGES_REQUESTED** [review]
One additional question

File: steps/appcenter-codepush-release-react-native/0.0.3/step.yml (hunk 1 above)
Comment (imrekel) [reply steps/appcenter-codepush-release-react-native/0.0.3/step.yml:47]: hey @adrianchinghc could you please mark this input as sensitive?

**Comment by trapacska** [comment]
@adrianchinghc could you please do the requested changes? We will need to close your PR within couple of days if the changes are not done unfortunately. 🙁

**Comment by adrianchinghc** [comment]
Hey everyone. The PR should be updated now with `is_sensitive`. Sorry for the delay. Was busy with work and didn't manage to look into this until yesterday.

**Review by sylank — APPROVED** [review]
LGTM 👍

**Pull request merged**